src/pretix/static/datetimepicker/bootstrap-datetimepicker.js

```diff
@@ -947,8 +947,17 @@
 
                 input.blur();
 
-                viewDate = date.clone();
+                // PROBLEM IS HERE 
+                // WHEN YOU CLICK OUT OF WIDGET WITHOUT SELECTING A DATE, date IS CURRENT DATE
+                // WHEN YOU SELECT DATE, date STILL JUNE 3 REGARDLESS OF SELECTED DATE
+                // THEREFORE, we do not want to set date to viewDate when clicking out of widget
 
+                // date.month() & viewDate.month() will be the same if date selected by user
+                // date.month() & viewDate.month() will be different if user clicks out of widget
+                // therefore, do not set date to be equal to viewDate if date.month() !== viewDate.month()
+                if (date.month() === viewDate.month()) {
+                    viewDate = date.clone();
+                }
```

Comment:
AFAICT that line does not help at all. And `if date.month() === viewDate.month()` there is no practical benefit to set the viewdate to a clone of the original date. So I would just remove the line completely:

```suggestion
                // CUSTOM CHANGE: when closing the picker without selecting a date
                // this would set the viewDate to basically now() as local var date does
                // no respect min- /max-date initially.
                // viewDate = date.clone();
```